modules/calc/calc.block.ini

```diff
@@ -0,0 +1,54 @@
+[.]
+entity: calc
+description: Position calc
+
+[INPA]
+type: pos_mux
+description: Position input A
+
+[INPB]
+type: pos_mux
+description: Position input B
+
+[INPC]
+type: pos_mux
+description: Position input C
+
+[INPD]
+type: pos_mux
+description: Position input D
+
+[A]
+type: param enum
+description: Sign invert input A
+0: value
+1: negvalue
+
+[B]
+type: param enum
+description: Sign invert input B
+0: value
+1: negvalue
+
+[C]
+type: param enum
+description: Sign invert input C
+0: value
+1: negvalue
+
+[D]
+type: param enum
+description: Sign invert input D
+0: value
+1: negvalue
+
+[FUNC]
+type: param enum
+description: Scale divisor after add
+0: summed
+1: summed_over_two
+2: summed_over_four
```

Comment:
Lets remove the enum restriction and let it be anything at all with no whitespace in it